src/node/url.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { default as urlUtil } from 'node-internal:url';
+import { ERR_MISSING_ARGS } from 'node-internal:internal_errors';
+
+const URL = globalThis.URL;
+const URLSearchParams = globalThis.URLSearchParams;
+
+function domainToASCII(domain: string): string {
+  if (arguments.length < 1) {
+    throw new ERR_MISSING_ARGS('domain');
+  }
+
+  return urlUtil.domainToASCII(`${domain}`);
+}
+
+function domainToUnicode(domain: string): string {
+  if (arguments.length < 1) {
+    throw new ERR_MISSING_ARGS('domain');
+  }
+
+  return urlUtil.domainToASCII(`${domain}`);
```

Comment:
shouldn't this to `urlUtil.domainToUnicode`?

Reply:
Yup